bugzilla2gitlab/models.py

```diff
@@ -118,21 +118,18 @@ def create_description(self, fields):
         self.description += markdown_table_row("Architecture", fields.get("rep_platform"))
 
         # add first comment to the issue description
-        if (fields["reporter"] == fields["long_desc"][0]["who"] and
-                fields["long_desc"][0]["thetext"]):
-            ext_description += "\n## Extended Description \n"
-            ext_description += "\n\n".join(re.split("\n*", fields["long_desc"][0]["thetext"]))
-            del fields["long_desc"][0]
-
         attachments = []
         to_delete = []
-        for i in range(0, len(fields["long_desc"])):
+        comment0 = fields["long_desc"][0]
+        if (fields["reporter"] == comment0["who"] and comment0["thetext"]):
+            ext_description += "\n## Extended Description \n"
+            ext_description += "\n\n".join(re.split("\n*", comment0["thetext"]))
+            self.update_attachments(fields["reporter"], comment0, attachments)
+            to_delete.append(0)
+
+        for i in range(1, len(fields["long_desc"])):
```

Comment:
This introduces a subtle bug for the situation that the first comment is not related to an attachment, but in fact a legitimate comment (i.e. the `if` loop condition above evaluates to `false`). Please revert to `del fields["long_desc"][0]` and start this loop from 0.

Reply:
Done